src/math.rs

```diff
@@ -34,32 +33,43 @@ pub fn lerp(source: f32, target: f32, weight: f32) -> f32 {
     source + weight * (target - source)
 }
 
-/// Remaps the value from one range to another. Return None if one of the
-/// ranges is infinite.
+/// Remaps the value from one range to another. Return None if conversion of the
+/// value type to f64 fails.
```

Comment:
Returning `None` for infinite ranges is still true I think. Let's keep that in the doc comment as well.